Makefile

```diff
@@ -139,7 +139,7 @@ endif
 	done && \
 	minikube ssh --profile ${MINIKUBE_PROFILE} 'sudo rm -rf /var/reana' && \
 	if [ $$(docker images | grep -c '<none>') -gt 0 ]; then \
-		docker images | grep '<none>' | awk '{print $$3;}' | xargs docker rmi; \
+		docker images | grep '<none>' | awk '{print $$3;}' | xargs docker rmi --force; \
```

Comment:
Hmm, I'm not sure we want to `--force`` here, since we could be deleting something that the user may not want to delete.

If this command fails because there are some images left in use, it is usually because the previous builds etc ended up badly.  

What I'm usually doing in these situations is:

```console
$ docker ps -a | grep -v k8s
```

and then I do `docker rm amazing_feynman` etc for all non-Kubernetes containers that left.

After this the `make` command passes well.

Perhaps that should do?

Reply:
Agree, have just encountered the same issue again and your solution works. :)